conda-env/dev.yml

```diff
@@ -1,22 +1,21 @@
 # Conda e3sm_to_cmip development environment
 name: e3sm_to_cmip_dev
 channels:
-  - conda-forge/label/nco_dev
   - conda-forge
   - defaults
 dependencies:
   # Base
   # ==================
-  - python=3.9.10
+  - python=3.9.13
   - pip=21.2.4
-  - nco=5.1.1alpha13
-  - cmor=3.6.1
-  - tqdm=4.62.3
+  - nco=5.1.1
+  - cmor=3.7.0
+  - tqdm=4.64.1
   - pyyaml=6.0
-  - xarray=0.21.1
-  - netcdf4=1.5.8
-  - dask=2022.1.1
-  - scipy=1.7.3
+  - xarray=2022.10.0
+  - netcdf4=1.6.1
+  - dask=2022.10.2
+  - scipy=1.9.3
```

Comment:
This PR has just been merged and includes `e3sm_to_cmip_dev` updates.

@TonyB9000 and @chengzhuzhang, I advise you to rebuild your `e3sm_to_cmip_dev` env with the latest `dev.yml` file.

Reply:
I interpret: "pull the master", delete the existing e2c_dev environment, re-create the e2c_dev environment with the given dev.yml file, then "pip install" the e2c into e2c_dev.

Reply:
I got:  Successfully installed e3sm-to-cmip-1.9.1rc1  (I assume this is right)